### PR TITLE
Feature/authorcard

### PR DESCRIPTION
**Pull request by kailasnadh790**
Issue: https://github.com/hlxsites/famous-smoke-cigaradvisor/issues/8

Test URLs: https://feature-authorcard--famous-smoke-cigaradvisor--hlxsites.hlx.page/cigaradvisor/drafts/Kailas/authorcard

### CODE REVIEW

File: head.html [hunk 1]
```diff
@@ -2,3 +2,4 @@
 <script src="/cigaradvisor/scripts/aem.js" type="module"></script>
 <script src="/cigaradvisor/scripts/scripts.js" type="module"></script>
 <link rel="stylesheet" href="/cigaradvisor/styles/styles.css"/>
+<link crossorigin="anonymous" rel="stylesheet" id="ca-alfaslabmonterrat-font-css" href="//fonts.googleapis.com/css?family=Alfa+Slab+One%7CMontserrat%3A300%2C400%2C500%2C600%2C700%2C800%2C900%7COpen+Sans%3A400%2C600%2C700&#038;ver=5.5.1" type="text/css" media="all" />
```
Comment (bstopp) [inline head.html:5]: this doesn't follow the best practices for loading fonts. See the docs on font fallback and and the use of the fonts.css

We can add this to the global styles.

**Review by bstopp — CHANGES_REQUESTED** [review]

File: head.html (hunk 1 above)
Comment (kailasnadh790) [reply head.html]: This is fixed.

Comment (Kronnox) [reply head.html]: Just stumbled upon this after i did this myself. Provided my integration off all the fonts in #47. Might make sense to sync on this. With my changes you should be able to just use `font-family: var(--ff-alfaslabone)`

File: cigaradvisor/blocks/authorcard/authorcard.css [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+    float: none;
+    width: 50%;
+    align-items: center;
+    justify-content: flex-start;
+    padding: 10px;
+}
+
+.authorcard.block .left-column h2 {
+    font-family: "alfa slab one", cursive;
+    font-size: 40px !important;
```
Comment (bstopp) [inline cigaradvisor/blocks/authorcard/authorcard.css:47]: We shouldn't need to use `!important` anywhere. 

We should evaluate what makes sense for consistency of font sizes for h1/h2/etc

Then we can have variables for header font sizes for s, s, m, l, xl - depending on how many and which ones we need.

File: cigaradvisor/blocks/authorcard/authorcard.css [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+    display: flex;
+    float: none;
+    width: 50%;
+    align-items: center;
+    justify-content: flex-start;
+    padding: 10px;
+}
+
+.authorcard.block .left-column h2 {
+    font-family: "alfa slab one", cursive;
```
Comment (bstopp) [inline cigaradvisor/blocks/authorcard/authorcard.css:46]: use the font family variable names

File: cigaradvisor/blocks/authorcard/authorcard.css [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+.authorcard.block .left-column h2:first-child {
+    font-size: 35px !important;
+    border-bottom: none;
+}
+
+
+.authorcard.block .left-column p {
+    font-family: "open sans", sans-serif;
+    font-weight: 600;
+    color: #141414;
```
Comment (bstopp) [inline cigaradvisor/blocks/authorcard/authorcard.css:65]: We should use variables for colors. Something meaningful that can be changed if necessary.

--gold, --bronze, etc.

**Review by bstopp — CHANGES_REQUESTED** [review]

File: cigaradvisor/blocks/authorcard/authorcard.css (hunk 3 above)
Comment (kailasnadh790) [reply cigaradvisor/blocks/authorcard/authorcard.css:46]: fixed

File: cigaradvisor/blocks/authorcard/authorcard.css (hunk 2 above)
Comment (kailasnadh790) [reply cigaradvisor/blocks/authorcard/authorcard.css:47]: fixed.

File: cigaradvisor/blocks/authorcard/authorcard.css (hunk 4 above)
Comment (kailasnadh790) [reply cigaradvisor/blocks/authorcard/authorcard.css]: fixed.

File: cigaradvisor/styles/styles.css
Comment (kailasnadh790) [inline cigaradvisor/styles/styles.css:1]: this file can be overridden with changes from header PR to resolve conflicts.

**Review by tmaret — DISMISSED** [review]

**Review by tmaret — APPROVED** [review]

**Review by tmaret — APPROVED** [review]

**Review by tmaret — APPROVED** [review]

**Pull request closed without merging**